src/bundle/Templating/Twig/ComponentExtension.php

```diff
@@ -51,12 +51,20 @@ public function getFunctions()
             new TwigFunction(
                 'ibexa_render_component_group',
                 [$this, 'renderComponentGroup'],
-                ['is_safe' => ['html']]
+                [
+                    'is_safe' => ['html'],
+                    'deprecated' => true,
```

Comment:
```suggestion
                    'deprecated' => '4.6',
```
I think it was possible to state version there.